libmscore/property.cpp

```diff
@@ -445,6 +445,8 @@ QVariant propertyFromString(Pid id, QString value)
                         return QVariant(int(GlissandoStyle::BLACK_KEYS));
                   else if ( value == "diatonic")
                         return QVariant(int(GlissandoStyle::DIATONIC));
+                  else if ( value == "portamento") 
+                        return QVariant(int(GlissandoStyle::PORTAMENTO));
                   else // e.g., normally "Chromatic"
```

Comment:
I guess this here is the reason why the upper case G doesn't matter, ? `Chromatic`/`GlissandoStyle::CHROMATIC` simply is the default.